### PR TITLE
add a reproducer header

### PR DESCRIPTION
**Pull request by IanButterworth**
Screenshot 2024-10-18 at 11 59 40 AM

Can be seen here: https://github.com/IanButterworth/SimpleLooper.jl/actions/runs/11406030258/job/31741980077?pr=5#step:6:43

cc. @Seelengrab

### CODE REVIEW

File: test_harness.jl [hunk 1]
```diff
@@ -8,6 +8,14 @@ kwargs = Kwargs.kwargs(; coverage=ENV["COVERAGE"],
                          test_args=ENV["TEST_ARGS"],
                          )
 
+println("""
+###############
+# To reproduce this CI run, check out <hash> of this repo and
```
Comment (Seelengrab) [inline test_harness.jl:13]:
```suggestion
# To reproduce this CI run locally, check out <hash> of this repo and
```

**Comment by IanButterworth** [comment]
CI here doesn't actually seem to run this action in its entirety... that should be fixed.

File: test_harness.jl [hunk 2]
```diff
@@ -7,6 +7,25 @@ kwargs = Kwargs.kwargs(; coverage=ENV["COVERAGE"],
...
+    commit = LibGit2.head(repo)
+    commit_sha = LibGit2.GitHash(commit)
+    commit_info = "commit $commit_sha of "
+catch e
+    @info "Could not determine commit of repo in current directory" exception=(e, catch_backtrace())
+end
+
+println("""
+###############
+# To reproduce this CI run locally, check out $(commit_info)this repo and
```
Comment (Seelengrab) [inline test_harness.jl:24]:
```suggestion
# To reproduce this CI run locally, check out $(commit_info) on this repo and
```

Comment (IanButterworth) [reply test_harness.jl:24]: See `commit_info` above. It's setup to handle if we can't determine the commit hash.

Comment (Seelengrab) [reply test_harness.jl]: Aah, nice - missed that!

**Review by giordano — APPROVED** [review]

**Pull request merged**